samples/simulation/qaoa/README.md

```diff
@@ -0,0 +1,85 @@
+---
+page_type: sample
+languages:
+- qsharp
+products:
+- qdk
+description: "Using QAOA to calculate the path in a weighted graph with the smallest cost"
+---
+
+# Quantum Approximate Optimization Algorithm
+
+This is an example of a Quantum Approximate Optimization Algorithm (QAOA) implemented in a Q# program. QAOA was first introduced by Farhi et al. in [A Quantum Approximate Optimization Algorithm](https://arxiv.org/abs/1411.4028).
+
+This sample is based on the "Traveling Santa" problem described by Stephen Jordan in his Quantum Algorithm Zoo post, [Traveling Santa Problem](http://quantumalgorithmzoo.org/traveling_santa/).
+
+## Prerequisites
+
+- The Microsoft [Quantum Development Kit](https://docs.microsoft.com/quantum/install-guide/).
+
+## Description
+
+Here we will use a combinatorial optimization problem to demonstrate the usage of an implementation of QAOA. We consider a "Traveling Santa" problem. Santa needs to visit a a few different houses and end up back on the North Pole. We can express this as a graph with 4 nodes, where each node represents a house and each node is connected to all other nodes. The edges in the graph each have a cost associated with them. The goal is to find the optimal route such that Santa visits all houses only once and ends up back on the North Pole.
+
+The idea behind QAOA is to express a problem as an Ising Hamiltonian and replace each variable z_j with a Pauli-Z operation acting on the jth qubit, such that
+
+H_C = \sum_ij J_ij Z_i Z_j + \sum_i h_i Z_i
```

Comment:
I think that GitHub and docs.microsoft.com/samples both don't support math notation in README.md; if we want to rely on heavier math, it may make sense to move this to being a Jupyter Notebook–based sample, perhaps?

Reply:
I've seen other code that uses latex in the docs, so I thought it would be acceptable here.

Reply:
On docs.microsoft.com/quantum, we've enabled MathJax usage to translate LaTeX to HTML on the client side. That same isn't enabled for github.com or docs.microsoft.com/samples, where this particular README gets published. It may be possible that other README files have LaTeX notation, of course, but I'd recommend against doing so given our current samples publication pipeline.

Reply:
Got it, thanks